contrib/endpoints/src/api_manager/api_manager_impl.cc

```diff
@@ -75,11 +85,41 @@ utils::Status ApiManagerImpl::Init() {
     global_context_->cloud_trace_aggregator()->Init();
   }
 
-  for (auto it : service_context_map_) {
-    if (it.second->service_control()) {
-      it.second->service_control()->Init();
-    }
+  if (service_context_map_.size() > 0) {
+    // Already initialized
+    return utils::Status::OK;
```

Comment:
this is wrong.   We have to call service_context->service_control()->Init() for each service_context. 

These sevice_context was created at master process.  (at line 61)